### ISSUE TITLE
Cannot set property 'onComponentDidUpdate' of undefined : builderbundle:55

### ISSUE THREAD

**Issue by x711Li** [ipselon/react-ui-builder#13]
Hey guys,

Started the project up with a react-bootstrap template. On load, none of the UI functionality works because of the following error:

Cannot set property 'onComponentDidUpdate' of undefined

**Comment by ipselon** [comment]
Sorry, couldn't understand such a behaviour. Could you tell what node version you are using, and your OS version ?

**Comment by x711Li** [comment]
Node version 0.12.7
Mac OS X 10.9.5

Thanks

**Comment by sinceow** [comment]
I got the same error on Mac OS X 10.11 with Node v0.12.4 =_=

**Comment by ipselon** [comment]
Also please tell what browser you are using.

**Comment by x711Li** [comment]
I'm on Google Chrome v43.0

**Comment by sinceow** [comment]
I'm on Google Chrome  42.0.2311.90 (64-bit)

**Comment by nibon** [comment]
I had but fixed this problem on ubuntu linux, node 0.12.7, npm 2.13.0 and various browser versions. When I tried to preview the gallery bootstrap project I got a EACCESS error on directory "/usr/local/lib/node_modules/". For some reason the node_modules npm and react-ui-builder hade owner nobody. Setting correct owner <code>sudo chown -R <username> /usr/local/lib/react-ui-builder</code> allowing my user write access, both the problem with preview and this was fixed.
